sources/Context.cpp

```diff
@@ -300,7 +303,8 @@ Context::execute() {
   simu_context->setWorkingDirectory(config_.outputDir().generic_string());
 
   switch (def_.simulationKind) {
-  case dfl::inputs::Configuration::SimulationKind::STEADY_STATE_CALCULATION: {
+  case dfl::inputs::Configuration::SimulationKind::STEADY_STATE_CALCULATION:
+  case dfl::inputs::Configuration::SimulationKind::STEADY_STATE_CALCULATION_AND_SECURITY_ANALYSIS: {
```

Comment:
I would prefer to not have STEADY_STATE_CALCULATION_AND_SECURITY_ANALYSIS here as we do not know if it is the first or the latter that is run